lib/topological_inventory/core/seeding.rb

```diff
@@ -0,0 +1,12 @@
+module TopologicalInventory
+  module Core
+    class Seeding
+      def self.seed!
+        SourceType.create_with(
+          :display_name => "Red Hat OpenShift Container Platform",
+          :vendor       => "Red Hat, Inc.",
+        ).find_or_create_by!(:name => "openshift")
```

Comment:
I don't like having pluggable things in this repo.

Reply:
Agree, but it would be good to get a demo schema in a comment or something to see what the intended use would be like.

Reply:
Okay i'll drop the seeding for now